msgraph-connector-test/src_test/com/axonivy/connector/office365/test/RestIvyTest.java

```diff
@@ -24,13 +23,11 @@ void beforeEach(AppFixture fixture) {
     fixture.config("RestClients.Microsoft 365 (Partial Graph API).Features",
             "ch.ivyteam.ivy.rest.client.mapper.JsonFeature");
     fixture.config("RestClients.Microsoft 365 (Partial Graph API).Url", "{ivy.app.baseurl}/api/graphMock");
-    // fixture.var("Twitter-connector.Url",
-    // "{ivy.app.baseurl}/api/twitterMock");
```

Comment:
remove?

Reply:
yes; the URI didn't match the connector under test. See the line just above where the real client is being changed.